lte/gateway/c/core/oai/tasks/amf/amf_client_servicer.cpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "include/amf_client_servicer.h"
+#include <memory>
+
+namespace magma5g {
+
+AmfClientServicer amf_client_servicer_g;
+
+// Fetch the amf client servicer reference
+AmfClientServicer& get_amf_client_server_ref() {
+  return amf_client_servicer_g;
+}
+
+// Initialize the client servicer layer
+void amf_client_servicer_init() {
+  auto authentication_client =
+      std::make_shared<AsyncM5GAuthenticationServiceClient>();
+
+  amf_client_servicer_g = AmfClientServicer(authentication_client);
```

Comment:
what is the significance of `_g`? is it to signify that this is grpc client? it does not help much to readability.

Reply:
ah this is to indicate that it is global?

Reply:
Yes..intention to denote it as global...This layer will serve as interface to all clients. After the merge it will be extended t o serve mobilityd and sessiond clients.